### ISSUE TITLE
Messagebus send isn't working

### ISSUE THREAD

**Issue by LinusS1** [MycroftAI/mycroft-core#2226]
## Software:
* Picroft
* 19.2.13

## Problem

This used to work before, but I've seen some work with refactoring the messagebus code, and that probably broke it.

This is basically my code:
```python
from mycroft.messagebus.send import send
send("skill.communications.device.new", {"message": "10.0.1.7"})
```
Which sends the message `skill.communications.device.new` from the code in my (communications) skill which handles new devices, to the Mycroft skill, to be registered.

Now it is failing with the error:
```python-traceback
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/home/pi/mycroft-core/.venv/lib/python3.5/site-packages/zeroconf.py", line 1423, in run
    handler(self.zc)
  File "/home/pi/mycroft-core/.venv/lib/python3.5/site-packages/zeroconf.py", line 1363, in <lambda>
    zeroconf=zeroconf, service_type=self.type, name=name, state_change=state_change
  File "/home/pi/mycroft-core/.venv/lib/python3.5/site-packages/zeroconf.py", line 1250, in fire
    h(**kwargs)
  File "/home/pi/mycroft-core/.venv/lib/python3.5/site-packages/zeroconf.py", line 1335, in on_change
    listener.add_service(*args)
  File "/opt/mycroft/skills/communications-skill.linuss1/shippingHandling.py", line 107, in add_service
    send_communication_to_messagebus("device", ip)
  File "/opt/mycroft/skills/communications-skill.linuss1/shippingHandling.py", line 26, in send_communication_to_messagebus
    send("skill.communications.{}.new".format(msg_type), {"message": "{}".format(str(msg))})
  File "/home/pi/mycroft-core/mycroft/messagebus/send.py", line 70, in send
    url = MessageBusClient.build_url(
AttributeError: type object 'MessageBusClient' has no attribute 'build_url'
```

**Comment by forslund** [comment]
Thanks for reporting, we'll get right on fixing this.

**Comment by forslund** [comment]
Resolved by #2230
